model-server-test/docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+services:
+  db:
+    build:
+      dockerfile: Dockerfile
+      context: docker
```

Comment:
I think we can get rid of the build step, which only takes time. The `initdb.sql` could simply be mounted to `/docker-entrypoint-initdb.d/initdb.sql` using a volume mount here in the compose file with the native postgres image.